src/test/java/core/basesyntax/service/RegistrationServiceImplTest.java

```diff
@@ -0,0 +1,112 @@
+package core.basesyntax.service;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.User;
+import org.junit.jupiter.api.BeforeAll;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class RegistrationServiceImplTest {
+    private static final User FIRST_VALID_TEST_USER = new User();
+    private static final User SECOND_VALID_TEST_USER = new User();
+    private static final User NULL_LOGIN_TEST_USER = new User();
+    private static final User NULL_PASSWORD_TEST_USER = new User();
+    private static final User SAME_FIRST_VALID_TEST_USER = new User();
+    private static final User INVALID_LOGIN_TEST_USER = new User();
+    private static final User INVALID_PASSWORD_TEST_USER = new User();
+    private static final User INVALID_AGE_TEST_USER = new User();
```

Comment:
User is not an immutable class so it cannot be constant (even you used setters to change it's value later)
Please use standardVariableNamingCase for them